### ISSUE TITLE
internationals units

### ISSUE THREAD

**Issue by tuxun** [nasa/NASA-3D-Resources#16]
in building instructions:
 [example](https://github.com/nasa/NASA-3D-Resources/blob/master/3D%20Printing/Curiosity%20Rover/Curiosity%20Rover%203D%20Printed%20Model/Detailed%20Curiosity%20Model%20(Large)/Detailed%20Curiosity%20Model%20(Large)%20-%20Build%20Instructions.pdf)

in part list we can see filenames are in millimeters, but the description of print bed is in inches. please use only metrics system, past proved than mix different units systems could lead to very bad things :/

**Comment by arc-webservices** [comment]
I completely agree, but the instructions are written by the various contributors and we have to accept whatever they send to us. Some of these contributors are no longer with NASA so getting updated instructions is no longer possible.

If the instructions are in a file easily editable then feel free to make the corrections and issue a pull request.
